### PR TITLE
class implementation indent fix

### PR DESCRIPTION
**Pull request by sbcgua**
to #337 fix class implementation indent

P.S. indentation code became less readable. I'd propose making the run non-static and split into 2 methods (well, the original one and the "fix" for class indentation). Can add a commit if OK.

### CODE REVIEW

**Comment by larshp** [comment]
yes please

**Comment by sbcgua** [comment]
done

File: src/abap/pretty_printer.ts [hunk 1]
```diff
@@ -32,15 +32,24 @@ class Stack {
...
+  private options: IndentationOptions;
+  private globalClasses = new Set();
+
+  constructor(file: ABAPFile, options: IndentationOptions) {
+    this.file = file;
+    this.options = options;
+  }
+
+  // returns list of expected indentation for each line/statement?
+  public run(): number[] {
```
Comment (larshp) [inline src/abap/pretty_printer.ts]: suggest moving `file` parameter from `constructor` to `run`

Comment (sbcgua) [reply src/abap/pretty_printer.ts]: done

**Comment by larshp** [comment]
1.6.18 published

**Comment by sbcgua** [comment]
🎉 Thanks !
Mockup loader is almost passing the build now ;)

**Comment by larshp** [comment]
just open more issues if more settings are needed 😄

**Pull request merged**